patternfly/patternfly-elements
# fix: reduce padding on the pfe jump link

## <Pull request description>

Updating padding for jump-links.
- https://rally1.rallydev.com/#/270861059696d/teamboard?detail=%2Fuserstory%2F505884023220&view=113b0e72-0781-41c7-b6d6-e161d3349c72
- https://xd.adobe.com/view/1a085703-7753-4fab-9fcf-d3e9683232b4-4f5b/
- https://github.com/patternfly/patternfly-elements/issues/1274
- Update variables to new standard with fallbacks in pfe-jump-links as well as pfe-accordion in order to get the override to work in jump-links for accordion.


### Preview 

Link(s) to demo page(s) where this element can be viewed:

- [pfe-accordion demo](https://deploy-preview-1482--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/)
- [pfe-jump-links demo](https://deploy-preview-1482--happy-galileo-ea79c4.netlify.app/elements/pfe-jump-links/demo/)

### What has changed and why



- pfe-jump-links.scss has been updated with new padding vars and values.
- pfe-accordion.scss has been updated with new vars so they can be overridden in jump-links.


### Testing instructions



1. Open the tabs demo page and you should see the following changes in the XD file 
2. Jump links panel should have no background color and no padding
3. Jump links panel accordion wrapped should have no padding

Sandbox page:
- https://web-dev-drupal-anbartle.dev.a1.vary.redhat.com/en/demo-web-components

Demo pages:
- https://deploy-preview-1482--happy-galileo-ea79c4.netlify.app/elements/pfe-jump-links/demo
- https://deploy-preview-1482--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/


XD files:
- https://xd.adobe.com/view/793e2f7d-a1e4-414e-a5da-26ab0d6013cd-2c6d/
- https://xd.adobe.com/view/1a085703-7753-4fab-9fcf-d3e9683232b4-4f5b/

Test style overrides via the `inspector-stylesheet`
paste in

for` jump-links`
```
:root {
    --pfe-accordion__panel-container--Padding: 100px;
    --pfe-accordion__panel--Padding: 100px;
    --pfe-jump-links--BackgroundColor: pink;
}
```
For `/elements/pfe-accordion/demo/`
```
:root {
    --pfe-accordion__panel-container--Padding: 100px;
    --pfe-accordion__panel--Padding: 100px;
}
```

Make sure overrides work and default styles are as expected from mockups.


#### Browser requirements

Your component should work in all of the following environments:



### Ready-for-merge Checklist





### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**
